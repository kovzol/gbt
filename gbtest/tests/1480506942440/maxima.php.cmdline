#!/bin/sh
cd tests/1480506942440
cat maxima.php.in | /usr/bin/timeout -k 120 60 /usr/bin/time /usr/bin/maxima > maxima.php.out 2>maxima.php.err