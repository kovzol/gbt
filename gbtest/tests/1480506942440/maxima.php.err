0.03user 0.01system 0:00.05elapsed 84%CPU (0avgtext+0avgdata 29420maxresident)k
0inputs+480outputs (0major+4177minor)pagefaults 0swaps
