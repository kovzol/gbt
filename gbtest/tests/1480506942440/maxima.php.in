load(grobner)$ poly_monomial_order:lex$ poly_reduction(poly_grobner([x^2+x*y-1,2*x*y+3*x,x^3],[x,y]),[x,y]);