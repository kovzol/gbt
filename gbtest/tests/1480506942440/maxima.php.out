
Maxima 5.37.2 http://maxima.sourceforge.net
using Lisp GNU Common Lisp (GCL) GCL 2.6.12
Distributed under the GNU Public License. See the file COPYING.
Dedicated to the memory of William Schelter.
The function bug_report() provides bug reporting information.
(%i1) (%o3)                                 [1]
(%i4) 